docs/src/example.md

```diff
@@ -1,88 +1,193 @@
-## jco Example Workflow
+## Jco Example Workflow
 
 Given an existing Wasm Component, `jco` provides the tooling necessary to work with this Component fully natively in JS.
 
-For an example, consider a Component `cowsay.wasm`:
+Jco also provides an experimental feature for generating components from JavaScript by wrapping [ComponentizeJS](https://github.com/bytecodealliance/ComponentizeJS) in the `jco componentize` command.
+
+To demonstrate a full end-to-end component, we can create a JavaScript component embedding Spidermnokey then run it in JavaScript.
+
+### Installing Jco
+
+Either install Jco globally:
 
 ```shell
-- cowsay.wasm
+$ npm install -g @bytecodealliance/jco
+$ jco --version
+1.0.3
 ```
 
-Where we would like to use and run this Component in a JS environment.
+Or install it locally and use `npx` to run it:
 
-### Inspecting Component WIT
+```shell
+$ npm install @bytecodealliance/jco
+$ npx jco --version
+1.0.3
+```
 
-As a first step, we might like to look instead this binary black box of a Component and see what it actually does.
+Local usage can be preferable to ensure the project is reproducible and self-contained, but requires
+replacing all `jco` shell calls in the following example with either `./node_modules/.bin/jco` or `npx jco`.
+
+### Installing ComponentizeJS
 
-To do this, we can use `jco wit` to extract the "WIT world" of the Component ([WIT](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md) is the typing language used for defining Components).
+To use ComponentizeJS, it must be separately installed, locally or globally depending on whether Jco was installed locally or globally:
```

Comment:
Nit: the order here suggests that "npm install -g" is the "locally" option. Also you say "or locally" below but there's no first part of that "or".

```suggestion
To use ComponentizeJS, it must be separately installed, globally or locally depending on whether Jco was installed globally or locally. Globally:
```